src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.Match.cs

```diff
@@ -100,7 +100,7 @@ public Match Match(string input, int startat)
             if (input == null)
                 throw new ArgumentNullException(nameof(input));
 
-            return Run(false, -1, input, 0, input.Length, startat);
+            return Run(false, -1, input, 0, input.Length, startat)!;
```

Comment:
NB: the `Run` method will only return null values in cases where the first parameter is set to `true`.